store/sqlstore/scheme_store.go

```diff
@@ -331,28 +355,33 @@ func (s *SqlSchemeStore) Delete(schemeId string) (*model.Scheme, error) {
 func (s *SqlSchemeStore) GetAllPage(scope string, offset int, limit int) ([]*model.Scheme, error) {
 	var schemes []*model.Scheme
 
-	scopeClause := ""
 	if scope != "" {
-		scopeClause = " AND Scope=:Scope "
+		if err := s.GetReplicaX().Select(&schemes, "SELECT * from Schemes WHERE DeleteAt = 0 AND Scope = ? ORDER BY CreateAt DESC LIMIT ? OFFSET ?", scope, limit, offset); err != nil {
+			return nil, errors.Wrapf(err, "failed to get Schemes")
+		}
+
+		return schemes, nil
 	}
 
-	if _, err := s.GetReplica().Select(&schemes, "SELECT * from Schemes WHERE DeleteAt = 0 "+scopeClause+" ORDER BY CreateAt DESC LIMIT :Limit OFFSET :Offset", map[string]interface{}{"Limit": limit, "Offset": offset, "Scope": scope}); err != nil {
+	if err := s.GetReplicaX().Select(&schemes, "SELECT * from Schemes WHERE DeleteAt = 0 ORDER BY CreateAt DESC LIMIT ? OFFSET ?", limit, offset); err != nil {
```

Comment:
Let's rewrite the query using squirrel and use it only in one place rather than splitting it into two.